apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -366,403 +316,347 @@ export const PoolAdvertisementPoster = ({
-                  defaultMessage: "Experience and education",
-                  id: "owzveI",
+                  defaultMessage:
+                    "These opportunities have the following requirements:",
+                  id: "t9Zy30",
                   description:
-                    "Title for experience and education pool requirements",
+                    "Lead-in text for other requirements on a pool advertisement",
                 })}
-              </IconTitle>
-              <div
-                data-h2-display="base(flex)"
-                data-h2-flex-direction="base(column) p-tablet(row)"
-                data-h2-align-items="base(center) p-tablet(stretch)"
-              >
-                <Card
-                  color="secondary"
-                  style={{ width: "100%" }}
-                  title={intl.formatMessage({
-                    defaultMessage: "Combination Experience",
-                    id: "7o+Vzu",
-                    description:
-                      "Title for pool applicant experience requirements",
-                  })}
-                >
-                  <Text>
-                    {intl.formatMessage({
-                      defaultMessage:
-                        "2 or more years of combined experience in a related field including any of the following:",
-                      id: "s60QyR",
-                      description:
-                        "lead in to list of experience required for a pool applicant",
-                    })}
-                  </Text>
-                  <ul>
-                    <li>
-                      {intl.formatMessage({
-                        defaultMessage: "On-the-job learning",
-                        id: "qNL/Rp",
-                        description:
-                          "pool experience requirement, on job learning",
-                      })}
-                    </li>
-                    <li>
-                      {intl.formatMessage({
-                        defaultMessage: "Non-conventional training",
-                        id: "YlWJ/N",
-                        description:
-                          "pool experience requirement, non-conventional training",
-                      })}
-                    </li>
-                    <li>
-                      {intl.formatMessage({
-                        defaultMessage: "Formal education",
-                        id: "DydUje",
+              </Text>
+              <DataRow
+                Icon={MapPinIcon}
+                label={intl.formatMessage({
+                  defaultMessage: "Location:",
+                  id: "H7M6Yv",
+                  description:
+                    "Label for pool advertisement location requirement",
+                })}
+                value={
+                  poolAdvertisement.isRemote
+                    ? intl.formatMessage({
+                        defaultMessage: "Remove optional",
+                        id: "14pmF7",
```

Comment:
typo

Reply:
Assuming the correct text is "Remote optional"?

Reply:
I think so

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/42d95f26-0eb0-4034-a5b5-38d6ca32be4a)

Reply:
Fixed in [fix remote typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6543/commits/c7494233c0045592cfde23ad99f24aa41ad73cf7)